integration-tests/e2e_tests_ci_tool/cmd/mask_test_config_cmd.go

```diff
@@ -0,0 +1,172 @@
+package cmd
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/olekukonko/tablewriter"
+	"github.com/pelletier/go-toml/v2"
+	"github.com/smartcontractkit/seth"
+	"github.com/spf13/cobra"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	ctf_config "github.com/smartcontractkit/chainlink-testing-framework/config"
+)
+
+var maskTestConfigCmd = &cobra.Command{
+	Use:   "mask-secrets",
+	Short: "Run 'echo ::add-mask::${secret}' for all secrets in the test config",
+	Run: func(_ *cobra.Command, _ []string) {
+		decoded, err := base64.StdEncoding.DecodeString(fromBase64TOML)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error decoding base64 TOML: %v\n", err)
+			os.Exit(1)
+		}
+		var config ctf_config.TestConfig
+		err = toml.Unmarshal(decoded, &config)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error unmarshalling base64 TOML: %v\n", err)
+			os.Exit(1)
+		}
+
+		showCmdInfo()
+
+		if config.ChainlinkImage != nil {
+			mustMaskSecret("chainlink image", safeDeref(config.ChainlinkImage.Image))
+			mustMaskSecret("chainlink version", safeDeref(config.ChainlinkImage.Version))
+		}
+		if config.Pyroscope != nil {
+			mustMaskSecret("pyroscope server url", safeDeref(config.Pyroscope.ServerUrl))
+			mustMaskSecret("pyroscope key", safeDeref(config.Pyroscope.Key))
+		}
+		if config.Logging != nil && config.Logging.Loki != nil {
+			// mustMaskSecret("loki tenant id", config.Logging.RunId)
+			mustMaskSecret("loki endpoint", safeDeref(config.Logging.Loki.Endpoint))
+			mustMaskSecret("loki tenant id", safeDeref(config.Logging.Loki.TenantId))
+			mustMaskSecret("loki basic auth", safeDeref(config.Logging.Loki.BasicAuth))
+			mustMaskSecret("loki bearer token", safeDeref(config.Logging.Loki.BearerToken))
+		}
+		if config.Logging != nil && config.Logging.Grafana != nil {
+			mustMaskSecret("loki grafana token", safeDeref(config.Logging.Grafana.BearerToken))
```

Comment:
`mustMaskSecret("loki grafana token"` -> mustMaskSecret("grafana bearer token"`? this has nothing to do with Loki, it's used only to access grafana API

Reply:
@Tofel this cmd was removed, no longer used.